pcsx2/CDVD/CdRom.cpp

```diff
@@ -751,20 +765,18 @@ void cdrWrite1(u8 rt)
 		break;
 		do_CdlPlay:
 		case CdlPlay:
-			// Taken from pcsxr
-			if (cdr.Reading)
-			{
-				StopReading();
-			}
 			if (cdr.SetlocPending)
 			{
 				memcpy(cdr.SetSectorSeek, cdr.SetSector, 4);
 				cdr.SetlocPending = 0;
 			}
+			// Need to figure out if data track or audio. Data tracks are treated as "00"
```

Comment:
Incorrect. Play with `track == 0` starts playing CDDA from the current hold position. The mechacon doesn't care whether it's data or audio for the purposes of the command.